q2_composition/assets/run_ancombc.R

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env Rscript
+
+# load libraries -----------------
+suppressWarnings(library(phyloseq))
+suppressWarnings(library(tidyverse))
+suppressWarnings(library(ANCOMBC))
+
+# load arguments -----------------
+cat(R.version$version.string, "\n")
+args=commandArgs(TRUE)
+
```

Comment:
look into if optparse is a better option here - want to remove this nasty list of command args if possible to prevent future bugs if/when params are modified